<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatchCode</class>
 <widget class="QMainWindow" name="PatchCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patch Code</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Write your custom code:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtDisasmPatch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnApplyPatch">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Patch</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkCompleteNops">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fill with NoOperation</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPatchWarn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
